app/main/server/preferences/pref-manager.js

```diff
@@ -1,43 +1,69 @@
 'use strict';
 
 const appPref = require('./app-pref');
+const themePref = require('./theme-pref');
+
 const APP_PREF_ID = 'Hain';
-const appPrefItem = {
-  id: APP_PREF_ID,
-  group: 'Application'
-};
+const THEME_PREF_ID = 'Themes';
+
+const appStaticPrefs = [
+  {
+    id: APP_PREF_ID,
+    group: 'Application'
+  },
+  {
+    id: THEME_PREF_ID,
+    group: 'Application'
+  }
+];
 
 module.exports = class PrefManager {
   constructor(workerProxy) {
     this.workerProxy = workerProxy;
     this.appPref = appPref;
+    this.themePref = themePref;
   }
   getPrefItems() {
     return this.workerProxy.getPluginPrefIds().then((pluginPrefIds) => {
       const pluginPrefItems = pluginPrefIds.map((id) => ({
         id,
         group: 'Plugins'
       }));
-      const prefItems = [appPrefItem].concat(pluginPrefItems);
+
```

Comment:
I think this and below empty lines are redundant changes

Reply:
Removed.